doc/tutorials/plugins.md

```diff
@@ -363,7 +363,8 @@ be called and the mounted file will be updated.
 
 We haven't discussed `ELEKTRA_SET_ERROR` yet. Because Elektra is a library, printing errors to stderr wouldn't be a good idea. Instead, errors
 and warnings can be appended to a key in the form of metadata. This is what `ELEKTRA_SET_ERROR` does. Because the parentKey always exists
-even if a critical error occurs, we append the error to the `parentKey`. The first parameter is an id specifying the general error that occurred.
+even if a critical error occurs, we append the error to the `parentKey`. If there are multiple errors in a configuration, only the first occurrence
```

Comment:
"append" is then actually the wrong word. The errors do not necessarily need to be in the configuration.

Reply:
I did not change this sentence you are actually referring to.  What about "we write the error to the `parentKey` if a critical error occurred"?